[Bug] sgl-kernel fails to compile FlashAttention on CUDA 12.8 with H20

### Checklist

- [x] I searched related issues but found no solution.
- [x] The bug persists in the latest version.
- [x] Issues without environment info and a minimal reproducible demo are hard to resolve and may receive no feedback.
- [x] If this is not a bug report but a general question, please start a discussion at https://github.com/sgl-project/sglang/discussions. Otherwise, it will be closed.
- [x] Please use English. Otherwise, it will be closed.

### Describe the bug

GPU: H20
CUDA: 12.8

I don’t have a physical machine, so I can’t compile using the officially provided Dockerfile; instead, I did it locally...

run `make build` compile sgl-kernel failed.
```
-- Found Python: /opt/conda/bin/python3 (found version "3.10.13") found components: Interpreter Development.Module Development.SABIModule
-- Detected CUDA_VERSION=12.8
CUDA_VERSION 12.8 >= 12.8
-- Found CUDAToolkit: /usr/local/cuda/include (found version "12.8.61")
-- PyTorch: CUDA detected: 12.8
-- PyTorch: CUDA nvcc is: /usr/local/cuda/bin/nvcc
-- PyTorch: CUDA toolkit directory: /usr/local/cuda
-- PyTorch: Header version is: 12.8
-- Found Python: /opt/conda/bin/python3 (found version "3.10.13") found components: Interpreter
CMake Warning at /opt/conda/lib/python3.10/site-packages/torch/share/cmake/Caffe2/public/cuda.cmake:140 (message):
  Failed to compute shorthash for libnvrtc.so
Call Stack (most recent call first):
  /opt/conda/lib/python3.10/site-packages/torch/share/cmake/Caffe2/Caffe2Config.cmake:86 (include)
  /opt/conda/lib/python3.10/site-packages/torch/share/cmake/Torch/TorchConfig.cmake:68 (find_package)
  CMakeLists.txt:42 (find_package)


-- USE_CUDNN is set to 0. Compiling without cuDNN support
-- USE_CUSPARSELT is set to 0. Compiling without cuSPARSELt support
-- USE_CUDSS is set to 0. Compiling without cuDSS support
-- USE_CUFILE is set to 0. Compiling without cuFile support
-- Autodetected CUDA architecture(s):  9.0 9.0 9.0 9.0 9.0 9.0 9.0 9.0
CMake Warning at /opt/conda/lib/python3.10/site-packages/torch/share/cmake/Caffe2/public/cuda.cmake:323 (message):
  pytorch is not compatible with `CMAKE_CUDA_ARCHITECTURES` and will ignore
  its value.  Please configure `TORCH_CUDA_ARCH_LIST` instead.
Call Stack (most recent call first):
  /opt/conda/lib/python3.10/site-packages/torch/share/cmake/Caffe2/Caffe2Config.cmake:86 (include)
  /opt/conda/lib/python3.10/site-packages/torch/share/cmake/Torch/TorchConfig.cmake:68 (find_package)
  CMakeLists.txt:42 (find_package)

....
....
....
[590/595] Building CUDA object CMakeFiles/flash_ops.dir/_deps/repo-flash-attention-src/hopper/instantiations/flash_fwd_hdim192_fp16_paged_split_softcapall_sm80.cu.o
[591/595] Building CUDA object CMakeFiles/flash_ops.dir/_deps/repo-flash-attention-src/hopper/instantiations/flash_fwd_hdim256_bf16_split_softcapall_sm80.cu.o
[592/595] Building CUDA object CMakeFiles/flash_ops.dir/_deps/repo-flash-attention-src/hopper/instantiations/flash_fwd_hdim256_fp16_split_softcapall_sm80.cu.o
ninja: build stopped: subcommand failed.

*** CMake build failed
  × Failed to build `/ossfs/workspace/offical_sglang/sglang/sgl-kernel`
  ├─▶ The build backend returned an error
  ╰─▶ Call to `scikit_build_core.build.build_wheel` failed (exit status: 1)
      hint: This usually indicates a problem with the package or the build environment.
make: *** [build] Error 2
```


--- 
ninja -v 2>&1 | tee /tmp/sgl-kernel-ninja.log
tail -n 200 /tmp/sgl-kernel-ninja.log

```bash
[4/6] /usr/local/cuda/bin/nvcc -forward-unknown-to-host-compiler -ccbin=/usr/bin/g++ -DPy_LIMITED_API=0x030a0000 -DUSE_C10D_GLOO -DUSE_C10D_NCCL -DUSE_DISTRIBUTED -DUSE_NVSHMEM -DUSE_RPC -DUSE_TENSORPIPE -Dflashmla_ops_EXPORTS -I/ossfs/workspace/offical_sglang/sglang/sgl-kernel/include -I/ossfs/workspace/offical_sglang/sglang/sgl-kernel/csrc -I/admin/tmp/_deps/repo-flashmla-src/csrc -I/admin/tmp/_deps/repo-flashmla-src/csrc/kerutils/include -I/admin/tmp/_deps/repo-flashmla-src/csrc/sm90 -I/admin/tmp/_deps/repo-flashmla-src/csrc/extension/sm90/dense_fp8 -I/admin/tmp/_deps/repo-flashmla-src/csrc/cutlass/include -I/admin/tmp/_deps/repo-flashmla-src/csrc/cutlass/tools/util/include -isystem /opt/conda/include/python3.10 -isystem /opt/conda/lib/python3.10/site-packages/torch/include -isystem /opt/conda/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /usr/local/cuda/targets/x86_64-linux/include -DONNX_NAMESPACE=onnx_c2 -Xcudafe --diag_suppress=cc_clobber_ignored,--diag_suppress=field_without_dll_interface,--diag_suppress=base_class_has_different_dll_interface,--diag_suppress=dll_interface_conflict_none_assumed,--diag_suppress=dll_interface_conflict_dllexport_assumed,--diag_suppress=bad_friend_decl --expt-relaxed-constexpr --expt-extended-lambda -D_GLIBCXX_USE_CXX11_ABI=1 -O3 -DNDEBUG -std=c++17 -Xcompiler=-fPIC -std=c++20 --expt-relaxed-constexpr --expt-extended-lambda --use_fast_math -Xcudafe=--diag_suppress=177 -gencode=arch=compute_90a,code=sm_90a -MD -MT CMakeFiles/flashmla_ops.dir/_deps/repo-flashmla-src/csrc/sm100/prefill/dense/fmha_cutlass_bwd_sm100.cu.o -MF CMakeFiles/flashmla_ops.dir/_deps/repo-flashmla-src/csrc/sm100/prefill/dense/fmha_cutlass_bwd_sm100.cu.o.d -x cu -c /admin/tmp/_deps/repo-flashmla-src/csrc/sm100/prefill/dense/fmha_cutlass_bwd_sm100.cu -o CMakeFiles/flashmla_ops.dir/_deps/repo-flashmla-src/csrc/sm100/prefill/dense/fmha_cutlass_bwd_sm100.cu.o
FAILED: [code=1] CMakeFiles/flashmla_ops.dir/_deps/repo-flashmla-src/csrc/sm100/prefill/dense/fmha_cutlass_bwd_sm100.cu.o 
/usr/local/cuda/bin/nvcc -forward-unknown-to-host-compiler -ccbin=/usr/bin/g++ -DPy_LIMITED_API=0x030a0000 -DUSE_C10D_GLOO -DUSE_C10D_NCCL -DUSE_DISTRIBUTED -DUSE_NVSHMEM -DUSE_RPC -DUSE_TENSORPIPE -Dflashmla_ops_EXPORTS -I/ossfs/workspace/offical_sglang/sglang/sgl-kernel/include -I/ossfs/workspace/offical_sglang/sglang/sgl-kernel/csrc -I/admin/tmp/_deps/repo-flashmla-src/csrc -I/admin/tmp/_deps/repo-flashmla-src/csrc/kerutils/include -I/admin/tmp/_deps/repo-flashmla-src/csrc/sm90 -I/admin/tmp/_deps/repo-flashmla-src/csrc/extension/sm90/dense_fp8 -I/admin/tmp/_deps/repo-flashmla-src/csrc/cutlass/include -I/admin/tmp/_deps/repo-flashmla-src/csrc/cutlass/tools/util/include -isystem /opt/conda/include/python3.10 -isystem /opt/conda/lib/python3.10/site-packages/torch/include -isystem /opt/conda/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /usr/local/cuda/targets/x86_64-linux/include -DONNX_NAMESPACE=onnx_c2 -Xcudafe --diag_suppress=cc_clobber_ignored,--diag_suppress=field_without_dll_interface,--diag_suppress=base_class_has_different_dll_interface,--diag_suppress=dll_interface_conflict_none_assumed,--diag_suppress=dll_interface_conflict_dllexport_assumed,--diag_suppress=bad_friend_decl --expt-relaxed-constexpr --expt-extended-lambda -D_GLIBCXX_USE_CXX11_ABI=1 -O3 -DNDEBUG -std=c++17 -Xcompiler=-fPIC -std=c++20 --expt-relaxed-constexpr --expt-extended-lambda --use_fast_math -Xcudafe=--diag_suppress=177 -gencode=arch=compute_90a,code=sm_90a -MD -MT CMakeFiles/flashmla_ops.dir/_deps/repo-flashmla-src/csrc/sm100/prefill/dense/fmha_cutlass_bwd_sm100.cu.o -MF CMakeFiles/flashmla_ops.dir/_deps/repo-flashmla-src/csrc/sm100/prefill/dense/fmha_cutlass_bwd_sm100.cu.o.d -x cu -c /admin/tmp/_deps/repo-flashmla-src/csrc/sm100/prefill/dense/fmha_cutlass_bwd_sm100.cu -o CMakeFiles/flashmla_ops.dir/_deps/repo-flashmla-src/csrc/sm100/prefill/dense/fmha_cutlass_bwd_sm100.cu.o
nvcc warning : incompatible redefinition for option 'std', the last value of this option was used
/admin/tmp/_deps/repo-flashmla-src/csrc/cutlass/include/cutlass/gemm/collective/sm100_blockscaled_mma_array_warpspecialized_rcggemm.hpp(694): warning #2908-D: the implicit by-copy capture of "this" is deprecated
          Tensor mSFB_tmp = observed_tma_load_sfb_->get_tma_tensor(shape(layout_SFB));
                            ^

Remark: The warnings can be suppressed with "-diag-suppress <warning-number>"

/admin/tmp/_deps/repo-flashmla-src/csrc/cutlass/include/cutlass/gemm/collective/sm100_blockscaled_sparse_mma_warpspecialized.hpp(902): warning #2908-D: the implicit by-copy capture of "this" is deprecated
          Tensor mSFB_tmp = observed_tma_load_sfb_->get_tma_tensor(shape(layout_SFB_));
                            ^

/admin/tmp/_deps/repo-flashmla-src/csrc/cutlass/include/cutlass/gemm/collective/sm100_blockscaled_mma_warpspecialized.hpp(717): warning #2908-D: the implicit by-copy capture of "this" is deprecated
          Tensor mSFB_tmp = observed_tma_load_sfb_->get_tma_tensor(shape(layout_SFB_));
                            ^

/admin/tmp/_deps/repo-flashmla-src/csrc/cutlass/include/cutlass/gemm/collective/sm100_blockscaled_mma_array_warpspecialized.hpp(727): warning #2908-D: the implicit by-copy capture of "this" is deprecated
          Tensor mSFB_tmp = observed_tma_load_sfb_->get_tma_tensor(shape(layout_SFB));
                            ^

/admin/tmp/_deps/repo-flashmla-src/csrc/cutlass/include/cutlass/gemm/collective/sm100_blockscaled_mma_mixed_tma_cpasync_warpspecialized.hpp(541): warning #2908-D: the implicit by-copy capture of "this" is deprecated
          Tensor mSFB_tmp = observed_tma_load_sfb_->get_tma_tensor(shape(layout_SFB_));
                            ^

/admin/tmp/_deps/repo-flashmla-src/csrc/cutlass/include/cutlass/gemm/collective/sm103_blockscaled_mma_warpspecialized.hpp(682): warning #2908-D: the implicit by-copy capture of "this" is deprecated
          Tensor mSFB_tmp = observed_tma_load_sfb_->get_tma_tensor(shape(params.layout_SFB));
                            ^

/admin/tmp/_deps/repo-flashmla-src/csrc/cutlass/include/cutlass/gemm/collective/sm103_blockscaled_mma_array_warpspecialized.hpp(810): warning #2908-D: the implicit by-copy capture of "this" is deprecated
          Tensor mSFB_tmp = observed_tma_load_sfb_->get_tma_tensor(shape(layout_SFB));
                            ^

/usr/local/cuda/targets/x86_64-linux/include/cuda/std/detail/libcxx/include/tuple: In substitution of ‘template<class ... _Tp> template<class _Tuple> template<class _Tuple2> using _PreferTupleLikeConstructor = cuda::std::__4::__tuple_constraints<_Tp>::__valid_tuple_like_constraints_rank_one<_Tuple>::_PreferTupleLikeConstructorImpl<_Tuple2> [with _Tuple2 = _Tuple; _Tuple = _Tuple; _Tp = {_Tp ...}]’:
/usr/local/cuda/targets/x86_64-linux/include/cuda/std/detail/libcxx/include/tuple:695:209:   required from here
/usr/local/cuda/targets/x86_64-linux/include/cuda/std/detail/libcxx/include/tuple:695:209: internal compiler error: Segmentation fault
  695 |     static constexpr bool __implicit_constructible =
      |                                                                                                                                                                                                                 ^
Please submit a full bug report,
with preprocessed source if appropriate.
See <http://bugzilla.redhat.com/bugzilla> for instructions.
Preprocessed source stored into /tmp/ccZm4jIe.out file, please attach this to your bugreport.
ninja: build stopped: subcommand failed.
```

### Reproduction

```bash
cd sglang/sgl-kernel
make build
```

### Environment
```bash
Python: 3.10.13 (main, Sep 11 2023, 13:44:35) [GCC 11.2.0]
CUDA available: True
GPU 0,1,2,3,4,5,6,7: NVIDIA H20-3e
GPU 0,1,2,3,4,5,6,7 Compute Capability: 9.0
CUDA_HOME: /usr/local/cuda
NVCC: Cuda compilation tools, release 12.8, V12.8.61
CUDA Driver Version: 570.133.20
PyTorch: 2.9.1+cu128
sglang: 0.5.9
sglang-kernel: 0.4.1.post1
flashinfer_python: 0.6.6
flashinfer_cubin: 0.6.6
flashinfer_jit_cache: Module Not Found
triton: 3.5.1
transformers: 5.3.0
torchao: 0.9.0
numpy: 1.26.4
aiohttp: 3.11.16
fastapi: 0.123.5
huggingface_hub: 1.7.1
interegular: 0.3.3
modelscope: 1.31.0
orjson: 3.10.16
outlines: 0.1.11
packaging: 25.0
psutil: 7.0.0
pydantic: 2.12.3
python-multipart: 0.0.20
pyzmq: 26.4.0
uvicorn: 0.34.0
uvloop: 0.21.0
vllm: 0.15.0
xgrammar: 0.1.27
openai: 2.6.1
tiktoken: 0.12.0
anthropic: 0.71.0
litellm: Module Not Found
decord2: 2.0.0
NVIDIA Topology: 
        GPU0    GPU1    GPU2    GPU3    GPU4    GPU5    GPU6    GPU7    NIC0    NIC1    NIC2    NIC3    CPU Affinity    NUMA Affinity   GPU NUMA ID
GPU0     X      NV18    NV18    NV18    NV18    NV18    NV18    NV18    NODE    NODE    SYS     SYS     0-47,96-143     0               N/A
GPU1    NV18     X      NV18    NV18    NV18    NV18    NV18    NV18    PIX     NODE    SYS     SYS     0-47,96-143     0               N/A
GPU2    NV18    NV18     X      NV18    NV18    NV18    NV18    NV18    NODE    NODE    SYS     SYS     0-47,96-143     0               N/A
GPU3    NV18    NV18    NV18     X      NV18    NV18    NV18    NV18    NODE    PIX     SYS     SYS     0-47,96-143     0               N/A
GPU4    NV18    NV18    NV18    NV18     X      NV18    NV18    NV18    SYS     SYS     PIX     NODE    48-95,144-191   1               N/A
GPU5    NV18    NV18    NV18    NV18    NV18     X      NV18    NV18    SYS     SYS     NODE    NODE    48-95,144-191   1               N/A
GPU6    NV18    NV18    NV18    NV18    NV18    NV18     X      NV18    SYS     SYS     NODE    PIX     48-95,144-191   1               N/A
GPU7    NV18    NV18    NV18    NV18    NV18    NV18    NV18     X      SYS     SYS     NODE    NODE    48-95,144-191   1               N/A
NIC0    NODE    PIX     NODE    NODE    SYS     SYS     SYS     SYS      X      NODE    SYS     SYS
NIC1    NODE    NODE    NODE    PIX     SYS     SYS     SYS     SYS     NODE     X      SYS     SYS
NIC2    SYS     SYS     SYS     SYS     PIX     NODE    NODE    NODE    SYS     SYS      X      NODE
NIC3    SYS     SYS     SYS     SYS     NODE    NODE    PIX     NODE    SYS     SYS     NODE     X 

Legend:

  X    = Self
  SYS  = Connection traversing PCIe as well as the SMP interconnect between NUMA nodes (e.g., QPI/UPI)
  NODE = Connection traversing PCIe as well as the interconnect between PCIe Host Bridges within a NUMA node
  PHB  = Connection traversing PCIe as well as a PCIe Host Bridge (typically the CPU)
  PXB  = Connection traversing multiple PCIe bridges (without traversing the PCIe Host Bridge)
  PIX  = Connection traversing at most a single PCIe bridge
  NV#  = Connection traversing a bonded set of # NVLinks

NIC Legend:

  NIC0: mlx5_bond_0
  NIC1: mlx5_bond_1
  NIC2: mlx5_bond_2
  NIC3: mlx5_bond_3


ulimit soft: 1048576
```

--- a/sgl-kernel/cmake/flashmla.cmake
+++ b/sgl-kernel/cmake/flashmla.cmake
@@ -17,6 +17,8 @@ set(FLASHMLA_CUDA_FLAGS
     "-Xcudafe=--diag_suppress=177"   # variable was declared but never referenced
 )
 
+set(FLASHMLA_ENABLE_SM100 OFF)
+
 # The FlashMLA kernels only work on hopper and require CUDA 12.4 or later.
 # Only build FlashMLA kernels if we are building for something compatible with
 # sm90a
@@ -29,6 +31,7 @@ if(${CUDA_VERSION} VERSION_GREATER 12.8)
     list(APPEND FLASHMLA_CUDA_FLAGS
         "-gencode=arch=compute_100a,code=sm_100a"
     )
+    set(FLASHMLA_ENABLE_SM100 ON)
 endif()
 if(${CUDA_VERSION} VERSION_GREATER_EQUAL "13.0")
     # Patch FlashMLA sources for SM103a support.
@@ -113,26 +116,30 @@ set(FlashMLA_SOURCES
     ${repo-flashmla_SOURCE_DIR}/csrc/sm90/prefill/sparse/instantiations/phase1_k576.cu
     ${repo-flashmla_SOURCE_DIR}/csrc/sm90/prefill/sparse/instantiations/phase1_k576_topklen.cu
 
-    # sm100 dense prefill/bwd.
-    ${repo-flashmla_SOURCE_DIR}/csrc/sm100/prefill/dense/fmha_cutlass_fwd_sm100.cu
-    ${repo-flashmla_SOURCE_DIR}/csrc/sm100/prefill/dense/fmha_cutlass_bwd_sm100.cu
-
-    # sm100 sparse prefill.
-    ${repo-flashmla_SOURCE_DIR}/csrc/sm100/prefill/sparse/fwd/head64/instantiations/phase1_k512.cu
-    ${repo-flashmla_SOURCE_DIR}/csrc/sm100/prefill/sparse/fwd/head64/instantiations/phase1_k576.cu
-    ${repo-flashmla_SOURCE_DIR}/csrc/sm100/prefill/sparse/fwd/head128/instantiations/phase1_k512.cu
-    ${repo-flashmla_SOURCE_DIR}/csrc/sm100/prefill/sparse/fwd/head128/instantiations/phase1_k576.cu
-    ${repo-flashmla_SOURCE_DIR}/csrc/sm100/prefill/sparse/fwd_for_small_topk/head128/instantiations/phase1_prefill_k512.cu
-
-    # sm100 sparse decode.
-    ${repo-flashmla_SOURCE_DIR}/csrc/sm100/decode/head64/instantiations/v32.cu
-    ${repo-flashmla_SOURCE_DIR}/csrc/sm100/decode/head64/instantiations/model1.cu
-    ${repo-flashmla_SOURCE_DIR}/csrc/sm100/prefill/sparse/fwd_for_small_topk/head128/instantiations/phase1_decode_k512.cu
-
     ${repo-flashmla_SOURCE_DIR}/csrc/extension/sm90/dense_fp8/dense_fp8_python_api.cpp
     ${repo-flashmla_SOURCE_DIR}/csrc/extension/sm90/dense_fp8/flash_fwd_mla_fp8_sm90.cu
     ${repo-flashmla_SOURCE_DIR}/csrc/extension/sm90/dense_fp8/flash_fwd_mla_metadata.cu
 )
+
+if(FLASHMLA_ENABLE_SM100)
+    list(APPEND FlashMLA_SOURCES
+        # sm100 dense prefill/bwd.
+        ${repo-flashmla_SOURCE_DIR}/csrc/sm100/prefill/dense/fmha_cutlass_fwd_sm100.cu
+        ${repo-flashmla_SOURCE_DIR}/csrc/sm100/prefill/dense/fmha_cutlass_bwd_sm100.cu
+
+        # sm100 sparse prefill.
+        ${repo-flashmla_SOURCE_DIR}/csrc/sm100/prefill/sparse/fwd/head64/instantiations/phase1_k512.cu
+        ${repo-flashmla_SOURCE_DIR}/csrc/sm100/prefill/sparse/fwd/head64/instantiations/phase1_k576.cu
+        ${repo-flashmla_SOURCE_DIR}/csrc/sm100/prefill/sparse/fwd/head128/instantiations/phase1_k512.cu
+        ${repo-flashmla_SOURCE_DIR}/csrc/sm100/prefill/sparse/fwd/head128/instantiations/phase1_k576.cu
+        ${repo-flashmla_SOURCE_DIR}/csrc/sm100/prefill/sparse/fwd_for_small_topk/head128/instantiations/phase1_prefill_k512.cu
+
+        # sm100 sparse decode.
+        ${repo-flashmla_SOURCE_DIR}/csrc/sm100/decode/head64/instantiations/v32.cu
+        ${repo-flashmla_SOURCE_DIR}/csrc/sm100/decode/head64/instantiations/model1.cu
+        ${repo-flashmla_SOURCE_DIR}/csrc/sm100/prefill/sparse/fwd_for_small_topk/head128/instantiations/phase1_decode_k512.cu
+    )
+endif()
 
 Python_add_library(flashmla_ops MODULE USE_SABI ${SKBUILD_SABI_VERSION} WITH_SOABI ${FlashMLA_SOURCES})
 target_compile_options(flashmla_ops PRIVATE
@@ -140,6 +147,9 @@ target_compile_options(flashmla_ops PRIVATE
     $<$<COMPILE_LANGUAGE:CUDA>:-std=c++20>
     $<$<COMPILE_LANGUAGE:CUDA>:${FLASHMLA_CUDA_FLAGS}>
 )
+if(FLASHMLA_ENABLE_SM100)
+    target_compile_definitions(flashmla_ops PRIVATE FLASHMLA_ENABLE_SM100)
+endif()
 target_include_directories(flashmla_ops PRIVATE
     ${repo-flashmla_SOURCE_DIR}/csrc
     ${repo-flashmla_SOURCE_DIR}/csrc/kerutils/include

--- a/sgl-kernel/csrc/flashmla_extension.cc
+++ b/sgl-kernel/csrc/flashmla_extension.cc
@@ -36,11 +36,13 @@ TORCH_LIBRARY_FRAGMENT(sgl_kernel, m) {
       "-> Tensor[]");
   m.impl("fwd_kvcache_mla", torch::kCUDA, &fwd_kvcache_mla);
 
+#ifdef FLASHMLA_ENABLE_SM100
   m.def(
       "dense_prefill_fwd(Tensor workspace_buffer, Tensor q, Tensor k, Tensor v, Tensor cumulative_seqlen_q, Tensor "
       "cumulative_seqlen_kv, Tensor o, Tensor lse, int mask_mode_code, float softmax_scale, int max_seqlen_q, int "
       "max_seqlen_kv, bool is_varlen) -> ()");
   m.impl("dense_prefill_fwd", torch::kCUDA, &FMHACutlassSM100FwdRun);
+#endif
 
   m.def("sparse_prefill_fwd(Tensor q, Tensor kv, Tensor indices, float sm_scale, int d_v) -> Tensor[]");
   m.impl("sparse_prefill_fwd", torch::kCUDA, &sparse_prefill_fwd);

